CMakeLists.txt

```diff
@@ -12,11 +12,11 @@ project(qBittorrent
 # use CONFIG mode first in find_package
 set(CMAKE_FIND_PACKAGE_PREFER_CONFIG ON)
 # version requirements
-set(requiredBoostVersion 1.40)
+set(requiredBoostVersion 1.65)
 set(requiredQtVersion 5.9.0)
-set(requiredOpenSSLVersion 1.0)
-set(requiredLibtorrentVersion 1.2.0)
-set(requiredZlibVersion 1.2.5.2)
+set(requiredOpenSSLVersion 1.1)
+set(requiredLibtorrentVersion 1.2.0...2.0)
```

Comment:
What this notation means?

Reply:
It means "from 1.2.0, inclusive, up to 2.0, inclusive".

Reply:
> Doesn't `set(requiredLibtorrentVersion 1.2.0)` mean any version starting from 1.2?

Yes, but that doesn't accurately reflect what we support as accurately as `1.2.0...2.0` does. If 2.1 is released tomorrow, we want qBittorrent to fail to build with it automatically, until we manually bump the supported version.

Reply:
How about 2.0.1, 2.0.2, etc.?

Reply:
I would expect specifying `2.0` means "any patch/tweak version, as long as major and minor are `2.0`"